packages/dapp/src/components/InvoicePDF.js

```diff
@@ -0,0 +1,529 @@
+    paddingBottom: 4,
+    marginBottom: 4,
+  },
+  tableContainer: {
+    flexDirection: 'row',
+    flexWrap: 'wrap',
+    marginTop: 24,
+    borderWidth: 1,
+    borderColor: '#D3D3D3',
+  },
+  container: {
+    flexDirection: 'row',
+    borderBottomColor: '#D3D3D3',
+    backgroundColor: '#D3D3D3',
+    borderBottomWidth: 1,
+    alignItems: 'center',
+    height: 24,
+    textAlign: 'center',
+    fontStyle: 'bold',
+    flexGrow: 1,
+  },
+  row: {
+    flexDirection: 'row',
+    borderBottomColor: 'black',
+    borderBottomWidth: 1,
+    alignItems: 'center',
+    height: 24,
+    fontStyle: 'bold',
+  },
+  listTitle: {
+    paddingLeft: 8,
+  },
+  listContainer: {
+    paddingLeft: 8,
+    border: 2,
+    paddingTop: 10,
+    marginTop: 5,
+  },
+  innerTitle: {
+    textAlign: 'center',
+  },
+  invisibleRow: {
+    flexDirection: 'row',
+    alignItems: 'center',
+    height: 20,
+    fontStyle: 'bold',
+  },
+  description: {
+    width: '50%',
+    textAlign: 'left',
+    borderRightColor: borderColor,
+    borderRightWidth: 1,
+    paddingLeft: 8,
+  },
+  totalDescription: {
+    width: '50%',
+    textAlign: 'left',
+    paddingLeft: 8,
+  },
+  amount: {
+    width: '45%',
+    textAlign: 'left',
+    paddingLeft: 10,
+  },
+  secondTitle: {
+    color: 'black',
+    letterSpacing: 4,
+    fontSize: 15,
+    textAlign: 'center',
+    textTransform: 'uppercase',
+    marginTop: '15',
+  },
+  multiDetailBlock: {
+    borderColor: '#D3D3D3',
+    border: '3',
+    marginTop: '10',
+  },
+  detailPair: {
+    textAlign: 'center',
+    paddingTop: 5,
+  },
+  detailRow: {
+    borderBottomColor: 'black',
+    borderBottomWidth: 1,
+    alignItems: 'center',
+    height: 24,
+    fontStyle: 'bold',
+  },
+});
+
+// font register for Rubik One
+const InvoicePDF = ({ invoice, symbol }) => {
+  const {
+    projectName,
+    projectDescription,
+    projectAgreement,
+    startDate,
+    endDate,
+    terminationTime,
+    client,
+    provider,
+    amounts,
+    network,
+    createdAt,
+    deposits,
+    releases,
+    disputes,
+    resolutions,
+  } = invoice;
+
+  //  TO BE REMOVED WHEN MERGED TO MAIN
+
+  //   const disputesDemo = [
+  //     {
+  //     sender: '0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266',
+  //     txHash:
+  //         '0xeef10fc5170f669b86c4cd0444882a96087221325f8bf2f55d6188633aa7be7c',
+  //     ipfsHash: 'QmbWqxBEKC3P8tqsKc98xmWNzrzDtRLMiMPL8wBuTGsMnR',
+  //     details: 'www.google.com',
+  //     timestamp: '14098404',
+  //     },
+  // ]
+
+  //   const resolutionsDemo = [
+  //     {
+  //     txHash:
+  //         '0x25c4670e7ceed732423670ee5fff4ceab52af92f541a2eacfd3b4e6377ab6729',
+  //     ipfsHash: 'QmbTh45HCTHISISNOTAREALHASHLMiMPBuTGsMnR',
+  //     clientAward: '1500000000000000000',
+  //     providerAward: '0',
+  //     resolutionFee: '5000000000000000',
+  //     resolverType: 'Arbitrator',
+  //     resolver: '0x63125c0d5Cd9071de9A1ac84c400982f41C697AE',
+  //     },
+  // ]
```

Comment:
What is this commented out section for? Should it be removed now or what is it doing?

Reply:
Just provides dummy info to view the layout the disputes/resolutions sections. Thought it might be useful to keep if more info is added/removed from those sections, but definitely should come out when merged to main.